### ISSUE TITLE
[example/customShortcut] milestones

### ISSUE THREAD

**Issue by zhoonit** [nnstreamer/nntrainer#342]
## basic functionality
- [x]  initiate project & add scaffolding
- [x] Add basic ui & router
- [x] Add drawing function (#341)

## trainer
- [x] attach nntrainer & sanity test (#360)
- [x] handle logic preparing trainer. (eg. save drawing to loadable format) (#369)
- [x] make model (*.ini) (#371)
- [x] run training (#371)

## inference
- [x] hand over test data to trainer (#416)
- [x] print result (After #416)

**Comment by zhoonit** [comment]
Need way to inference. So this is postponed

**Comment by zhoonit** [comment]
> Need way to inference. So this is postponed

Change user scenario that does not need inference api (#416)
